### ISSUE TITLE
LSTM模型尝试结果

### ISSUE THREAD

**Issue by NiyunZhou** [NiyunZhou/The21-dayExpendables#32]
### 昨天试了一下 LSTM相关的模型，结构是 
Frame level data (300 frames * 1024 feature) 
-> 分三份 (100frames * 3 * 1024 feature)
-> 分别Batch Normalization 
-> 分别max pooling (20 frames * 3 * 1024 feature) 
-> Concat (60 frames * 3 * 1024 feature) 
-> LSTM (1024 cells * 2 layers, unroll 60 iters) last state (1024) 
->Batch Normalization  
->dropout (0.5) 
-> FC (~4200 classes) 

### 结构图如下：
![image](https://cloud.githubusercontent.com/assets/14039749/26336028/920859c8-3fa2-11e7-9324-1e9f5cd96f49.png)
### Loss：（learning rate 5e-5）
![image](https://cloud.githubusercontent.com/assets/14039749/26336084/fa07cd60-3fa2-11e7-9ce8-ad24d1bb713d.png)
基本在1.2个epoch左右就稳定了（横坐标是steps，一个epoch大概38000个step）
### training set准确率如下：
![image](https://cloud.githubusercontent.com/assets/14039749/26336102/2bd95142-3fa3-11e7-9072-bdf632421821.png)
GAP在0.7左右波动，跟video不会差太多。
不知道是参数的问题，还是LSTM对Feature的效果有限？ @LC1332  请求导师指点:stuck_out_tongue_closed_eyes:

@SophieGuan 的在#31 提到，其他人尝试LSTM的结果也不是很好。

另外，我昨天又检查了一下数据集，发现一些这样的特点：youtube上的视频前20s左右会有一些outline或者简介，这个部分的内容比较diverse。会有各种文字，PS的图片，各种吸引眼球的片段等。之后的部分会比较稳定，是视频的主要内容。我在考虑使用稍微偏后面的frame level feature是否会更有利于判断？
